server/src/test/java/io/seata/server/coordinator/DefaultCoordinatorTest.java

```diff
@@ -220,27 +222,42 @@ public static void afterClass() throws Exception {
         for (GlobalSession globalSession : globalSessions) {
             globalSession.closeAndClean();
         }
-
-        SessionHolder.destroy();
     }
 
     @AfterEach
-    public void tearDown() {
+    public void tearDown() throws IOException {
+        destroySessionHolder();
         deleteDataFile();
     }
 
-    private static void deleteDataFile() {
+    private static void deleteDataFile() throws IOException {
         File directory = new File(sessionStorePath);
         File[] files = directory.listFiles();
-        for (File file : files) {
-            file.delete();
+        if (files != null && files.length > 0) {
+            for (File file : files) {
+                Files.delete(Paths.get(file.getPath()));
+            }
         }
     }
+
     private static void deleteAndCreateDataFile() throws IOException {
+        destroySessionHolder();
         deleteDataFile();
         SessionHolder.init(StoreMode.FILE.name());
     }
 
+    private static void destroySessionHolder() {
+        try {
+            SessionHolder.getRootSessionManager();
```

Comment:
This should be `SessionHolder.destroy` to determine whether or not null, SessionHolder as a static method class, there is no guarantee that the internal initialization.

Reply:
yeah,now add null check in SessionHolder.destroy()